packages/orchestrator/lib/clients/types.ts

```diff
@@ -18,7 +18,6 @@ interface SyncArgs {
         environment_id: number;
         connection_id: string;
     };
-    syncJobId: number;
```

Comment:
`syncJobId` is actually not necessary. SyncJob will be created/updated at runtime.